packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -127,16 +130,29 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
   }
 }
 
+String getCmakeArch({targetPlatform: TargetPlatform})
+{
+  switch(targetPlatform) {
+    case TargetPlatform.windows_x64:
+      return "x64";
+    case TargetPlatform.windows_arm64:
+      return "ARM64";
+  }
+  throwToolExit("We should be building for windows.");
+}
```

Comment:
Nit:

```suggestion
String _getCmakeArch(TargetPlatform targetPlatform) {
  switch (targetPlatform) {
    case TargetPlatform.windows_x64:
      return 'x64';
    case TargetPlatform.windows_arm64:
      return 'ARM64';
    default:
      throwToolExit('Unsupported target platform "$targetPlatform".');
  }
}
```

Reply:
Good! That's done.

Reply:
Re-opening as the latest change doesn't follow the style guide.